### ISSUE TITLE
arista:eos miscategorized as cisco:ios 

### ISSUE THREAD

**Issue by chipzzz** [splunk/splunk-connect-for-syslog#2139]
Similar to https://github.com/splunk/splunk-connect-for-syslog/issues/1578

Running version : 2.49.7

Running `arista_eos` and `cisco_ios` also have the `Cisco Networks Add-on` splunk app installed 

-- Shown without headers --

Working correctly:
```
SuperServer: %SYS-5-CLI_SCHEDULER_ENABLED: CliScheduler is enabled, continuing its execution of scheduled CLI jobs.
tp: %SPANTREE-6-INTERFACE_STATE: Interface Ethernet29/1 instance MST0 moving from learning to forwarding
```

NOT working correctly:
```
%AGENT-6-INITIALIZED: Agent 'AleL3Agent-primary' initialized; pid=XXX
%PROCMGR-6-PROCESS_STARTED: 'StrataCounters' starting with PID=XXX (PPID=XXXX) -- execing '/usr/bin/StrataCounters'
```

**Comment by mstopa-splunk** [comment]
@chipzzz can you please send a pcap file or a few raw messages for tests?

**Comment by mstopa-splunk** [comment]
fixed in v3.4.0: https://github.com/splunk/splunk-connect-for-syslog/pull/2146#issuecomment-1706513992 , thank you for reporting
